CHANGELOG.md

```diff
@@ -81,6 +83,14 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
 - **CUMULUS-2518**
   - Update sf-event-sqs-to-db-records to not throw if a collection is not
     defined on a payload that has no granules/an empty granule payload object
+- **CUMULUS-2512**
+  - Updated ingest package S3 provider client to take additional parameter
+    `remoteAltBucket` on `download` method to allow for per-file override of
+    provider bucket for checksum
+  - Updated @cumulus/ingest.fetchTextFile's signature to be parameterized and
+    added `remoteAltBucket`to allow for an override of the passed in provider
+    bucket for the source file
+  - Update "eslint-plugin-import" to be pinned to 2.22.1
```

Comment:
We seem to be missing this part of the changelog
```
### Changed

- Removed "node-forge", "xml-crypto" from audit whitelist, added "underscore"
  per https://github.com/nasa/cumulus/pull/2216
```

Reply:
And the `audit-ci.json` file changes
```
    "whitelist": ["underscore"]
```

Reply:
That was deliberate as those changes were made in the main branch in a different PR.  The backport PR included them because they hadn't been pulled back to v8

Reply:
Explicitly PR #2216

Reply:
Ah gotcha